assets/app/styles/_flexbox.less

```diff
@@ -132,4 +132,182 @@
      -moz-align-self: @align;
       -ms-align-self: @align;
           align-self: @align;
-}
\ No newline at end of file
+}
+
+
+
+
+// Generic layout attr utils, inspired by Angular Material/Polymer flex layouts
```

Comment:
Just checked, the Grunt autoprefixer task runs when ./hacks/build-assets.sh is run, but not with ./hacks/serve-local-assets.sh because autoprefixer isn't run as part of grunt watch (which is run in grunt build).  So the vendor prefixes are added on a full build, but not during local development.  

I'm tabling this flex stuff for now anyway, but its possible we want to tinker with autoprefixer a bit.